ocaml/networkd/bin/network_monitor_thread.ml

```diff
@@ -42,8 +42,7 @@ let send_bond_change_alert _dev interfaces message =
   let ifaces = String.concat "+" (List.sort String.compare interfaces) in
   let module XenAPI = Client.Client in
   let session_id =
-    XenAPI.Session.login_with_password ~rpc:xapi_rpc ~uname:"" ~pwd:""
-      ~version:"1.4"
+    XenAPI.Session.login_with_password ~rpc ~uname:"" ~pwd:"" ~version:""
```

Comment:
the version parameter is ignored, has been since the first commit, so I doubt it's ever going to be useful
https://github.com/xapi-project/xen-api/blob/6b0ddfe5430f5f588dc7b226ce966434d099a4e9/ocaml/xapi/xapi_session.ml#L728